Makefile

```diff
@@ -38,16 +38,6 @@ clean:
 	rm -rf $(OUT_DIR)
 
 proto-gen:
-	protoc --proto_path=$(PROTO_DIR) \
-		--proto_path=third_party/proto \
-		--go_out=$(PROTO_OUT_DIR) \
-			--go_opt=paths=import \
-			--go_opt=module=$(MODULE) \
-		--go-grpc_out=$(PROTO_OUT_DIR) \
-			--go-grpc_opt=paths=import \
-			--go-grpc_opt=module=$(MODULE) \
-		--grpc-gateway_out=$(PROTO_OUT_DIR) \
-			--grpc-gateway_opt logtostderr=true \
-			--grpc-gateway_opt=paths=import \
-			--grpc-gateway_opt=module=$(MODULE) \
-		$(PROTO_DIR)/panacea_oracle/*/*/*.proto
\ No newline at end of file
+	cd $(PROTO_DIR) && buf generate; cd -
```

Comment:
I think we need to execute script like below:
Because if we add deps in `buf.yaml` it must execute to `buf mod update`
```
buf mod update
```

Reply:
Good point: 514d017e6d05e82ec4527ac28fc99b24d35e3764
Later, we need to pin dependencies so that you don't update some dependencies unintentionally.
https://docs.buf.build/tour/add-a-dependency#pin-your-dependencies